wattwatchers_rest_api_v3_long_energy.ipynb

```diff
@@ -410,28 +411,28 @@
         "    print('no devices associated with the API key')\n",
         "    return []\n",
         "  return load_long_energy_for_devices(devices, energy_params)"
-      ],
-      "execution_count": null,
-      "outputs": []
+      ]
     },
     {
       "cell_type": "markdown",
+      "metadata": {
+        "id": "Aehrt0BUI1RE"
+      },
       "source": [
         "# Convenience methods to convert dates to timestamps\n",
         "\n",
         "The API methods above use timestamps and time intervals. \n",
         "Below is a convenience method to convert two date and time strings to Unix UTC timestamp and interval to pass into the API methods.\n"
-      ],
-      "metadata": {
-        "id": "Aehrt0BUI1RE"
-      }
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "iO0dXSWLASYh"
+      },
+      "outputs": [],
       "source": [
-        "import datetime as dt\n",
-        "import pytz\n",
-        "\n",
         "# Convenience method to convert a start date and end date string\n",
         "# into a end time stamp and interval that can be used as parameters for the API methods.\n",
         "# Start and end date parameters are to be formatted as '2021-06-29 17:08:00' (or according tot he date_format parameter passed in)\n",
```

Comment:
Typo `to the`

Reply:
I just had the thought, perhaps this should be the https://en.wikipedia.org/wiki/ISO_8601#:~:text=As%20of%20ISO%208601-1,minute%20between%2000%20and%2059. format, which is the same as you have but the space is replaced by `T`